tfac/MRSA_filtering.py

```diff
@@ -0,0 +1,17 @@
+import pandas as pd
+import numpy as np 
+
+def export_mrsa_inputfile(OutputPath):
+    """Select top 10% genes per component and export csv file to run clusterProfiler in R"""
+    d = pd.read_csv('tfac/data/mrsa/cmtf_components_genes.csv')
```

Comment:
@JacksonLChin can help to hook this right into the factorization output rather than a file.